openpype/hosts/houdini/api/pipeline.py

```diff
@@ -53,7 +53,7 @@ def install():
     pyblish.api.register_host("hpython")
 
     pyblish.api.register_plugin_path(PUBLISH_PATH)
-    register_loader_plugin_path(LOAD_PATH)
+    register_loader_plugins_path(LOAD_PATH)
     avalon.api.register_plugin_path(LegacyCreator, CREATE_PATH)
```

Comment:
Are we sure this was originally a typo? :) The surrounding functions line 55 and 57 also don't have the plural on `plugin`?

Reply:
> The surrounding functions line 55 and 57 also don't have the plural on plugin?

I would say that it is because it is registering a path for a plugin class, but `register_loader_plugins_path` is registering path for loader plugins. Both variants are acceptable for me.

Reply:
> I would say that it is because it is registering a path for a plugin class, but register_loader_plugins_path is registering path for loader plugins. Both variants are acceptable for me.

Aren't they both just registering a path from where plug-ins get loaded? I'd stick to the naming conventions that already exist if the function intend to do the same? In this case we might be somewhat biased because the other 'naming conventions' come from Pyblish and Avalon API - but still I wouldn't see why we wouldn't match for consistency.